pkg/scrape/scrape.go

```diff
@@ -468,12 +468,12 @@ mainLoop:
 
 			tl := sl.target.Labels()
 			tl = append(tl, labels.Label{Name: "__name__", Value: profileType})
-			for _, l := range sl.externalLabels {
+			sl.externalLabels.Range(func(l labels.Label) {
```

Comment:
For reference:
https://github.com/prometheus/prometheus/blob/2d588725c12e46e4e71da3899ec05bb933151021/model/labels/labels.go#L426